I05038-1-README_cn.md

```diff
@@ -0,0 +1,95 @@
+[English](./README.md) | 简体中文
```

Comment:
Change the name of the file to "README_cn.md". Also for all other files should follow the same pattern i.e. <FILENAME>_cn.md, where <FILENAME> is the existing name in the English version.